src/logger/index.ts

```diff
@@ -1,23 +1,20 @@
 import winston from 'winston';
 import { consoleFormat } from 'winston-console-format';
-import { z } from 'zod';
 
-export const logFormatSchema = z.union([z.literal('json'), z.literal('pretty')]);
+export const logFormatOptions = ['json', 'pretty'] as const;
```

Comment:
A bit unfortunate is that we don't have validation functions for the individual logger fields (e.g. `logFormat`) but checking them is not that hard (use array includes or typeof check for bools).